test/KetlAttestation.deploy.ts

```diff
@@ -0,0 +1,185 @@
+import { BigNumber, providers } from 'ethers'
+import { DEV_KETL_ATTESTATION_CONTRACT } from '@big-whale-labs/constants'
+import { KetlAttestation, KetlAttestation__factory } from '../typechain'
+import { ethers, upgrades } from 'hardhat'
+import { expect } from 'chai'
+import { getFakeAttestationCheckerVerifier } from './utils/fakes'
+import {
+  getLegacyMintCalldata,
+  getLegacyRegisterEntanglementCalldata,
+  getLegacyTokenHolders,
+} from '../scripts/helpers'
+
+describe('KetlAttestation Deploy Tests', () => {
+  it('should be able to deploy KetlAttestation and populate data correctly', async function () {
+    const uri = 'https://game.example/api/item/{id}.json'
+    const version = '0.0.1'
+
+    const realProvider = new providers.JsonRpcProvider(
+      process.env.ETH_RPC,
+      'maticmum'
```

Comment:
In the context of this test, the only way the test would pass if this value was `maticmum`, so I think it makes sense to specify it here instead of getting it from somewhere else.